functions-methods.js

```diff
@@ -33,4 +55,26 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+console.log("Opdracht 3");
+
+function checkEmailValidity(email) {
+    switch (true) {
+        case (email.includes("@") === false):
+            return false;
+        case (email.includes(",") === true):
+            return false;
+        case (email.lastIndexOf(".") === email.length -1):
+            return false;
+        default:
+            return true;
+    }
```

Comment:
Het is niet gebruikelijk om een statische waarde te gebruiken als waarde in de switch. Het is in deze situatie ook niet nodig. Ik zou zelf dit mooier en beter vinden
```suggestion
    return email.includes("@") && !email.includes(",") && email.lastIndexOf(".") !== email.length -1;
    }
```